src/frontend/components/Accordion/Accordion.js

```diff
@@ -0,0 +1,27 @@
+import "./Accordion.css";
+import React from "react";
+import { useState } from "react";
+import AccordionInput from "../AccordionInput/AccordionInput";
+
+function Accordion({ data }) {
+  const [bool, setBool] = useState(false);
```

Comment:
`bool` seems like a generalized name. Maybe you could name it in a way that others understand what this state actually does.

Reply:
Yes right let me change it